components/infobox/extensions/commons/metadata_generator.lua

```diff
@@ -0,0 +1,172 @@
+local Template = require('Module:Template')
+local Games = require('Module:Games')
+local Variables = require('Module:Variables')
+local StringUtils = require('Module:StringUtils')
+local Class = require('Module:Class')
+
+local MetadataGenerator = {}
+
+function MetadataGenerator.tournament(args)
+	if String.isEmpty(args.publisherdescription) then
+		error('You must provide the publisherdescription param!')
+	end
+
+	local output
+	local frame = mw.getCurrentFrame()
+
+	local name = not String.isEmpty(args.name) and (args.name):gsub('&nbsp;', ' ') or mw.title.getCurrentTitle()
+
+	local type = args.type
+	local locality = Localisation.getLocalisation({displayNoError = true}, args.country)
+
+	local organizers = {
+		args['organizer-name'] or args.organizer,
+		args['organizer2-name'] or args.organizer2,
+		args['organizer3-name'] or args.organizer3,
+		args['organizer4-name'] or args.organizer4,
+		args['organizer5-name'] or args.organizer5
+	}
+
+	local tier = args.liquipediatier and Template.safeExpand(frame, 'TierDisplay', {args.liquipediatier}) or nil
+
+	if tier then
+		tier = (tonumber(
+			Template.safeExpand(frame, 'TierDisplay/number', {args.liquipediatier})
+		) or 0) > 4 and tier:lower() or tier
+	else
+		tier = 'Unknown Tier'
+	end
+
+	local tierType = (tier == 'qualifier' or tier == 'showmatch') and tier or 'tournament'
+	local publisher = Variables.varDefault(args.publisherdescription, '')
+	local date, tense = MetadataGenerator.getDate(args.edate or args.date, args.sdate)
+
+	local teams = args.team_number
+	local players = args.player_number
+
+	local game = (not String.isEmpty(args.game) and args.game ~= 'csgo') and Games.abbr[args.game]
+
+	local prizepoolusd = args.prizepoolusd and ('$' .. args.prizepoolusd .. ' USD') or nil
+	local prizepool = prizepoolusd or (
+		args.prizepool and args.localcurrency and (
+			Variables.varDefault('localcurrencysymbol', '') .. args.prizepool ..
+			Variables.varDefault('localcurrencysymbolafter', '') .. ' ' ..
+			Variables.varDefault('localcurrencycode', '')
+		)
+	)
+	local charity = args.charity == 'true' and true
+	local dateVerb = (tense == 'past' and 'took place ') or (tense == 'future' and 'will take place ') or 'takes place '
+	local dateVerbPublisher =
+		(tense == 'past' and ' which took place ') or
+		(tense == 'future' and ' which will take place ') or
+		' taking place '
+
+	output = StringUtils.interpolate('${name} is a${type}${locality}${game}${charity}${tierType}${organizer}', {
+		name = name,
+		type = type and ('n ' .. type:lower() .. ' ') or '',
+		locality = locality and (locality .. ' ') or '',
+		game = game and (game .. ' ') or '',
+		charity = charity and 'charity ' or '',
+		tierType = tierType,
+		organizer = organizers[1] and (' organized by ' .. organizers[1]) or ''
+	})
+
+	if organizers[2] then
+		output = output .. (organizers[3] and ', ' or ' and ') ..
+			organizers[2] .. (organizers[3] and (', and ' .. organizers[3]) or '') .. '. '
+	else
```

Comment:
in lines 33-37 we set more organizers, so we should add the option here too (else those additional organizers(4&5) up there are useless
maybe something like:
```suggestion
	if organizers[2] then
		local organizersExceptStartAndEnd = Table.copy(copy)
		table.remove(organizersExceptStartAndEnd)
		table.remove(organizersExceptStartAndEnd, 1)
		output = output .. table.,concat(organizersExceptStartAndEnd, ', ')
			.. ' and ' .. organizers[3] .. '.'
	else
```